ui/src/app/admin/@tab/manage-person/page.tsx

```diff
@@ -0,0 +1,20 @@
+import { managePersonResults } from '@/lib/fetchers';
+import PersonTable from '@/app/admin/_components/personTable';
+import { memberAttributeResults } from '@/lib/actions';
+
+const PersonTab = async (searchParams) => {
+    const searchUid = searchParams.searchParams.searchUid;
+    const groupingsInfo = await managePersonResults(searchUid);
+    const userInfo = searchUid === undefined ? undefined : (await memberAttributeResults([searchUid])).results[0];
```

Comment:
Let's call these variables managePersonResults and memberAttributeResults so it's more clear what exactly we are passing into the PersonTable.